dataactvalidator/validation_handlers/validator.py

```diff
@@ -150,19 +143,23 @@ def validate(record,rules,csvSchema,fileType,interfaces):
                 # If type failed, don't do value checks
                 continue
             #Field must pass all rules
+            # temporary fix for long/short colnames until legacy validation rules are removed:
+            # use long colnames in the record header
+            colMapping = interfaces.validationDb.getShortToLongColname()
+            recordLong = {colMapping.get(key,key):value for key, value in record.items()}
             for currentRule in ruleSubset :
                 if(checkRequiredOnly and currentRule.rule_type_id != interfaces.validationDb.getRuleType("REQUIRED_CONDITIONAL")):
                     # If data is empty, only check conditional required rules
                     continue
-                if(not Validator.evaluateRule(currentData,currentRule,currentSchema.field_type.name,interfaces,record)):
+                if(not Validator.evaluateRule(currentData,currentRule,currentSchema.field_type.name,interfaces,recordLong)):
                     recordFailed = True
                     failedRules.append([fieldName,"".join(["Failed rule: ",str(currentRule.description)]), currentData, str(currentRule.original_label)])
         # Check all multi field rules for this file type
         multiFieldRules = interfaces.validationDb.getMultiFieldRulesByFile(fileType)
         for rule in multiFieldRules:
-            if not Validator.evaluateRule(record,rule,None,interfaces,record):
+            if not Validator.evaluateRule(recordLong,rule,None,interfaces,recordLong):
                 recordFailed = True
-                failedRules.append(["MultiField", "".join(["Failed rule: ",str(rule.description)]), Validator.getMultiValues(rule, record, interfaces), str(rule.original_label)])
+                failedRules.append(["MultiField", "".join(["Failed rule: ",str(rule.description)]), Validator.getMultiValues(rule, recordLong, interfaces), str(rule.original_label)])
```

Comment:
I think what we had decided previously was to stick the 702's acceptance criteria of keeping the long fieldname in the error reports. I suspect when agencies start using the short names they will request error reports that reflect their submissions.